phpstan.neon.dist

```diff
@@ -9,6 +9,7 @@ parameters:
 		- tests/
 	bootstrapFiles:
 		- plugins/speculation-rules/load.php
+		- plugins/webp-uploads/load.php
```

Comment:
Just to know why we need to add load file here?

Reply:
It's for PHPStan to know about the main file constant.